app/src/main/kotlin/com/simplemobiletools/calculator/operation/FactorialOperation.kt

```diff
@@ -0,0 +1,21 @@
+package com.simplemobiletools.calculator.operation
+
+import com.simplemobiletools.calculator.operation.base.Operation
+import com.simplemobiletools.calculator.operation.base.UnaryOperation
+
+class ParOperation(value: Double) : UnaryOperation(value), Operation {
+
+    override fun getResult(): Double {
+        var result = 1.0
+        if (value==0.0 || value==1.0){
+            return result
+        } else {
+            var base: Int = value.toInt()
+            for (i in 1..base){
+                //Find Factorial of a number using for loop
```

Comment:
remove this comment, fix spaces, remove type declaration from "base", it isnt needed in such cases